src/logging.cc

```diff
@@ -786,7 +794,7 @@ inline void LogDestination::LogToSinks(LogSeverity severity,
                                        size_t message_len) {
   ReaderMutexLock l(&sink_mutex_);
   if (sinks_) {
-    for (int i = sinks_->size() - 1; i >= 0; i--) {
+    for (ptrdiff_t i = sinks_->size() - 1; i >= 0; i--) {
```

Comment:
This can be right. `sinks_` is of type `std::vector<LogSink*>` whose `size` method returns a type compatible to `size_t`. So the proposed should be just fine.